openlibrary/templates/type/i18n/edit.html

```diff
@@ -78,19 +77,19 @@
 
             $ xnamespace = namespace or "/"
 
-            $for ns in _.get_namespaces():
+            $for ns in i18n.get_namespaces():
```

Comment:
Where is get_namespaces() defined? I'm not finding it.

Reply:
I noticed this too, and am surprised it is not raising errors somewhere, but because of my investigation above where I conclude that this particular i18n feature is deprecated and not being taken advantage of anywhere, I wasn't so concerned since the screenshot evidence shows no other reported errors and a clear improvement in display.

I'd say next steps are to get agreement that this feature is no longer needed, and remove it, rather than attempt to reverse engineer the details of how it used to work.

Reply:
Actually these functions are from infogami https://github.com/infogami/infogami/blob/master/infogami/utils/i18n.py#L27

If we're removing this, we shouldn't bother making cosmetic changes to get it to render and should just go ahead and remove it (IMO).

Although the specific bug report was about rendering, the real bug was "I can't edit the Dutch translation" which this PR won't address.

Reply:
Thanks @tfmorris for locating those in infogami, I was wondering how it wan't causing errors.

A Dutch translation can be undertaken by creating a `openlibrary/i18n/nl` dir and copying the `messages.po` , following the instructions at https://github.com/internetarchive/openlibrary/tree/master/openlibrary/i18n

... I think we've established that the process as currently documented isn't as user-friendly as it could be  though :( It is quite technical and involved a lot of coder and github familiarity, which are not necessarily skills that a good translator should be expected to have.

I'm not sure what the next steps are. I was hoping someone could take up some translations and identify some clear improvements we could make. I'm not so familiar with how translators like to operate, and (perhaps unfortunately!) I can navigate the possibly awkward data formats and github commits relatively easily, so that doesn't slow me down ... I'm much more blocked by not knowing any Dutch ;)